src/banana/data/db.py

```diff
@@ -55,7 +55,8 @@ class Theory(Base):
     Qmb = Column(Float)
     Qmc = Column(Float)
     Qmt = Column(Float)
-    Qref = Column(Float)
+    QrefQED = Column(Float)
+    Qedref = Column(Float)
```

Comment:
Another tiny remark: you changed the wrong one xD
```suggestion
    Qref = Column(Float)
    QrefQED = Column(Float)
```